src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/FeatureFlagController.java

```diff
@@ -37,6 +54,34 @@ public ResponseEntity<Object> getFeatureFlag(@RequestParam(value = "flagName") S
 
     }
 
+    @GetMapping(
+        path = ALL_FEATURE_FLAG_STATUSES_URI,
+        produces = V1.MediaType.MAP_ASSIGNMENTS,
+        consumes = {"application/json"}
+    )
+    @ResponseStatus(code = HttpStatus.OK)
+    @Operation(summary = "get the statuses of all feature flags",
+        security =
+            {
+            @SecurityRequirement(name = AUTHORIZATION),
+            @SecurityRequirement(name = SERVICE_AUTHORIZATION)
+            })
+    @ApiResponse(
+        responseCode = "200",
+        description = "OK",
+        content = @Content(schema = @Schema(implementation = Object.class))
+    )
+    @ApiResponse(
+        responseCode = "400",
+        description = V1.Error.INVALID_REQUEST,
+        content = @Content()
+    )
```

Comment:
I don;t think there is a 400 response possile for this API as there is no validation required.